GetandTakeAPI/Program.cs

```diff
@@ -0,0 +1,98 @@
+using Autofac;
```

Comment:
Please, check my project configuration [here](https://github.com/dotnet-fizzyy/ASP.Net-Identity/blob/master/src/IdentityWebApi/Program.cs#L6) how to split your configuration for ASP.Net app (both MVC and Web API), it will reduce extra size or `Program.cs` and make your configuration easier and more reusable